.dagger/README.md

```diff
@@ -74,26 +74,13 @@ Connect to it from a dagger cli:
 
 In core/schema, changes utilizing the dagql package modify the engine's GraphQL API. API documentation and SDK bindings must be generated and committed when modifying the schema. See "Docs" and "SDKs" below for more granular generation functionality. This command also runs go generate for engine code.
 
-    dagger call generate export --path=.
+    dagger call generate -o .
 
 > [!NOTE]
 >
 > For `PHP` and `Elixir` SDKs it's important to manually delete the generated folder before running
 > this command
 
-
-## Docs
-
-Lint the docs:
-
-    dagger call docs lint
-
-Auto-generate docs components:
-
-    dagger call docs generate -o .
-
-    
-
```

Comment:
not sure if this was meant to be included here?

Reply:
yep, seems like `dagger call docs` is not a thing anymore.  Has been removed sometime ago and we forgot to update the README back then

Reply:
It's actually `dagger call -m docs` now (recent change)